setup.cfg

```diff
@@ -5,5 +5,5 @@ requires = python-twisted
 post-install = distro/redhat/misc/postinstall
 
 [install]
-prefix = $WORKSPACE/graphite
-install-lib = $WORKSPACE/graphite
+prefix = $WORKSPACE/.build/virtualenv
```

Comment:
This is worthy of a comment noting the customization is very krux-specific, and is for ve-packager running under Jenkins.